File: Python Excel Examples/WorksheetsApiDemo/updateWorksheetZoom.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration
from spirecloudexcel.models import Color

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.worksheets_api.WorksheetsApi(configuration)

name = "UpdateWorksheetZoom.xlsx"
folder = "ExcelDocument";
storage = ""
sheetName = "Sheet1"
value = 150
api.update_worksheet_zoom(name, sheetName, value, folder=folder, storage=storage)

File: PHP Excel Examples/WorksheetsApiDemo/updateWorksheetZoom.php
<?php

use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new WorksheetsApi($configuration);

$name = "UpdateWorksheetZoom.xlsx";
$sheetName = "Sheet2";
$value = 150;
$folder = "input";
$storage = null;
$result = $apiInstance->updateWorksheetZoom($name, $sheetName, $value, $folder, $storage);